app/Services/MagazineService.php

```diff
@@ -28,9 +28,9 @@ public function getMagazineByShortName($shortName)
      *
      * @return \Illuminate\Database\Eloquent\Collection
      */
-    public function getAllMagazines()
+    public function getAllMagazines($order = 'Asc')
     {
-        return $this->magazineRepo->find();
+        return $this->magazineRepo->find($order);
     }
 
     /**
```

Comment:
@aabi01 Did you tested it?

Default order is 'Asc' that is 'oldest on top'. Shouldn't we also need to pass parameter $order as `desc` from controller?